src/main/flight/pid.h

```diff
@@ -218,10 +218,11 @@ typedef struct pidProfile_s {
     uint8_t simplified_master_multiplier;
     uint8_t simplified_roll_pitch_ratio;
     uint8_t simplified_i_gain;
-    uint8_t simplified_pd_ratio;
-    uint8_t simplified_pd_gain;
+    uint8_t simplified_d_gain;
+    uint8_t simplified_pi_gain;
     uint8_t simplified_dmin_ratio;
     uint8_t simplified_feedforward_gain;
+    uint8_t simplified_pitch_pi_gain;
```

Comment:
Please follow [the rules for changing parameter groups](https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md).

Reply:
Whoops, sorry, will fix.